lib/graph.ts

```diff
@@ -298,6 +274,23 @@ class GraphImpl implements Graph, Graph.Transformer {
           throw new Error(`multiple nodes output to one data value: ${dataIndex}`);
         }
         this._allData[dataIndex]._from = i;
+
+        // for the 'Constant' operator, just create a new edge in the graph corresponding to the 'output' of the
+        // operator and ignore the node from the graph
+        if (nodeProto.opType === 'Constant') {
+          if (!nodeProto.attribute || nodeProto.attribute.length !== 1 || !nodeProto.attribute[0].t) {
+            throw new Error(`missing attributes or missing tensor value in attributes for this Constant operator`);
+          }
+          if (!nodeProto.output || nodeProto.output.length !== 1) {
+            throw new Error(`missing output or incorrect number of outputs for this Constant operator`);
+          }
+          node.outputs.pop();
+          node.executeNode = false;
+
+          const tensor = Tensor.fromProto(nodeProto.attribute[0].t);
+          this._allData[dataIndex]._from = -1;
+          this._allData[dataIndex].tensor = tensor;
```

Comment:
Line 292 can directly contain Tensor.fromProto(nodeProto.attribute[0].t);. There is no 'if'...'else' like there was previously